src/v1/sources/shopify/pixelUtils.js

```diff
@@ -57,7 +81,7 @@ const createMessage = (eventType, eventName, properties, context) => {
 
 const pageViewedEventBuilder = (inputEvent) => {
   const { data, context } = inputEvent;
-  const pageEventContextValues = mapObjectKeys(context, contextualFieldMappingJSON);
+  const pageEventContextValues = mapContextLineObjectKeys(context, contextualFieldMappingJSON);
   const message = new Message(INTEGERATION);
```

Comment:
can reuse createMessage function

Reply:
thanks, did a small change in createMessage to accomodate page calls too.